src/witan/models/run_models.clj

```diff
@@ -102,10 +103,26 @@
       (data-csv/write-csv out-file
                           (convert-row-maps-to-vector-of-vectors ordered-data)))))
 
+(defn get-district [gss-code]
```

Comment:
I was thinking of def-ing the reading of the edn file so we don't read it each time we do a lookup.
Something like:
```Clojure
(def local-authorities
    (:gss-code-to-district 
         (clojure.edn/read-string (slurp "resources/default_datasets/local_authorities.edn"))))
```